examples/spring-boot-with-source-manifests-example/src/main/java/io/dekorate/examples/sbsourcemanifests/Main.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2018 The original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.dekorate.examples.sbsourcemanifests;
+
+import io.dekorate.option.annotation.GeneratorOptions;
+import io.dekorate.prometheus.annotation.EnableServiceMonitor;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.RestController;
+
+@GeneratorOptions(inputPath = "source-manifests")
+@EnableServiceMonitor
```

Comment:
With a quick glimpse this annotation doesn't have a purpose in this example, so I'd rather have it removed.